### PR TITLE
integrationtest_ros_scanner_node.cpp: minor fixes

### PR DESCRIPTION
**Pull request by martiniil**
- remove magic number
- reduce number of spinning threads

### CODE REVIEW

File: test/integration_tests/integrationtest_ros_scanner_node.cpp [hunk 1]
```diff
@@ -131,7 +133,7 @@ int main(int argc, char* argv[])
   ros::init(argc, argv, "integrationtest_ros_scanner_node");
   ros::NodeHandle nh;
 
-  ros::AsyncSpinner spinner{ 2 };
+  ros::AsyncSpinner spinner{ 1 };
```
Comment (agutenkunst) [inline test/integration_tests/integrationtest_ros_scanner_node.cpp:137]: Maybe remove at all?

File: test/integration_tests/integrationtest_ros_scanner_node.cpp [hunk 2]
```diff
@@ -44,6 +44,8 @@ static constexpr int QUEUE_SIZE{ 10 };
 
 static const std::string LASER_SCAN_RECEIVED{ "LASER_SCAN_RECEIVED" };
 
+static constexpr double DEFAULT_X_AXIS_ROTATION{ 2.3998277 };  // corresponds to 137.5 degree
```
Comment (agutenkunst) [inline test/integration_tests/integrationtest_ros_scanner_node.cpp]: could you use `degreeToRad` ?

**Review by agutenkunst — CHANGES_REQUESTED** [review]

File: test/integration_tests/integrationtest_ros_scanner_node.cpp (hunk 2 above)
Comment (martiniil) [reply test/integration_tests/integrationtest_ros_scanner_node.cpp]: Done.

File: test/integration_tests/integrationtest_ros_scanner_node.cpp (hunk 1 above)
Comment (martiniil) [reply test/integration_tests/integrationtest_ros_scanner_node.cpp:137]: The `SubscriberMock subscriber` needs at least 1 spinning thread.

**Pull request merged**